apps/examples/wc-angular12/src/app/app.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule
+  ],
+  providers: [],
+  bootstrap: [AppComponent],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
It seems that Angular will only understand how to deal with Web Components after the `CUSTOM_ELEMENTS_SCHEMA` is added. Isn't it worth of mention in the README?

Reply:
very good point